### DISCUSSION TITLE
Google calendars shared with me

### DISCUSSION THREAD

**Discussion by sghalliday** [phw198/OutlookGoogleCalendarSync#1967] [Q&A]
I have OGCS setup and running.  It syncs all my outlook events to my Google calendar so that my wife can see them.  However, even though I have it setup to do two way syncs, I cannot see any of the events in her calendar in my outlook.  She has google setup to share her calendar with my google calendar and that works fine. I see them if I look directly at my google calendar, but cannot see them in outlook.  Is there something I have missed?  Is there a way to see  a shared google calendar in my outlook?

..steve

**Comment by phw198** [comment]
You'd obviously need to select her calendar as the source Google calendar in OGCS to have it sync to Outlook. You've only described syncing from "my Outlook to my Google calendar"...

**Reply by sghalliday** [reply]
OK, thanks.  I assumed that as her stuff was visible in my calendar it should come across.  I will do as you say.  Thanks again

**Reply by phw198** [reply]
Just to say, as it sounds like you need to sync multiple calendars (both yours and hers), you'll very likely be wanting to create a second Settings _Profile_ in OGCS.
